selenium/src/test/resources/reportng-1.1.4/CHANGELOG.txt

```diff
@@ -0,0 +1,57 @@
+Changes in version 1.1.4
```

Comment:
Not clear what this reporting directory is used for.  Its unusual for jars to be found in the resources directory for example.   Has this been accidently included - or were you using it for command line report generation or something?